packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -13,8 +13,8 @@
 # limitations under the License.
 
 Name:           ecs-init
-Version:        1.4.0
-Release:        2%{?dist}
+Version:        1.5.0
+Release:        1%{?dist}
```

Comment:
Yes.  Our convention is to have the `Version` match the Agent's version and to number the `Release` starting at 1.